metricbeat/scripts/docs_collector.py

```diff
@@ -177,17 +178,21 @@ def collect(beat_name):
     module_list_output = generated_note
 
     module_list_output += '[options="header"]\n'
-    module_list_output += '|========================\n'
-    module_list_output += '|Modules   |Metricsets   \n'
+    module_list_output += '|===================================\n'
+    module_list_output += '|Modules   |Metricsets   |Dashboards  \n'
 
     for key, m in sorted(six.iteritems(modules_list)):
 
         release_label = ""
         if m["release"] != "ga":
             release_label = m["release"] + "[]"
 
-        module_list_output += '|{} {}   |{}    \n'.format("<<metricbeat-module-" +
-                                                          key + "," + m["title"] + ">> ", release_label, "")
+        dashboards = "no"
+        if m["dashboards"]:
+            dashboards = "yes"
```

Comment:
`dashboards = "yes" if m["dashboards"] else "no"`?

Reply:
Works for me. I will wait with the change until we get some feedback from @dedemorton on if we could use something else then yes/no like a checkmark / emoji. 

Added the `in progress` label.